handlers/server_spec_handler.go

```diff
@@ -4,10 +4,10 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
+	"strings"
 
 	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
 
-	"github.com/manifoldco/promptui"
```

Comment:
We don't need this library, please run `go mod tidy` after the change to remove it from deps.

Reply:
@uds5501 I went ahead and removed this unused package.